examples/customresources/apis/proto/api/component/gizmo/v1/gizmo.proto

```diff
@@ -26,6 +26,12 @@ service GizmoService {
     };
   }
 
+  rpc DoWriteDataFile(DoWriteDataFileRequest) returns (DoWriteDataFileResponse) {
```

Comment:
I'm hesitant about expanding the API for gizmo. I suppose it's somewhat interesting to show that modular resources can write to files, but I wonder if we could use the `testmodule`'s `helper` resource and some new custom command to test that writes to files happen in the expected directory. Do you have any thoughts @Otterverse ?

Reply:
I debated this as well. I liked Gizmo because it seemed like a better full e2e test. However, I was on the fence and `testmodule` would mean changing fewer LOC which is always a good thing.

Reply:
Yeah I'd vote for using `testmodule` instead of expanding the gizmo API if that doesn't sound like too much work.

Reply:
You got it. Switching now.

Reply:
Agree with benji.

Reply:
Resolving.